pkg/stub/config.go

```diff
@@ -221,6 +221,10 @@ func (h *Handler) buildFileMaps(cfg *v1.Config, forceRebuild bool) error {
 		cm = &corev1.ConfigMap{}
 		cm.Name = util.IST2ImageMap
 		cm.Namespace = v1.OperatorNamespace
+		if cm.Annotations == nil {
```

Comment:
Just a nit, but you could move this one and the one inside of the `if err == nil` block right under the `cm, err := h.configmapclientwrapper.Get(util.IST2ImageMap)` to de-duplicate.

Reply:
but the `Get` will return a nil for `cm` though, right? ... at least I believe that is what I observed

that said, I don't need the nil check when I'm constructing the cm from scratch here ... so I will at least get rid of that